ckanext/security/mailer.py

```diff
@@ -36,10 +42,14 @@ def send_reset_link(user):
 def _build_footer_content(extra_vars):
     custom_path = config.get('ckanext.security.brute_force_footer_path')
     if (custom_path and os.path.exists(custom_path)):
-        log.warning('Overriding brute force lockout email footer with {}'.format(custom_path))
+        log.warning('Overriding brute force lockout email footer with %s',
+                    custom_path)
         with open(custom_path, 'r') as footer_file:
             footer_content = footer_file.read()
-        env = config['pylons.app_globals'].jinja_env
+        if tk.check_ckan_version('2.9'):
+            env = flask.current_app.jinja_env
```

Comment:
CKAN 2.8 has a mixture of Flask and Pylons. Will that affect how `env` is used?

Reply:
Possibly... We haven't had any capacity to test under 2.8 as yet. Could possibly look at changing this to use a helper to check if the current request is flask or pylons